template/package.json

```diff
@@ -12,6 +12,7 @@
     "jest": "^24.8.0",
     "jest-transform-stub": "^2.0.0",
     "parcel-bundler": "^1.6.1",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
while this was discussed in the previous PR, the linting fails by not having this package... So I've included it here... This may need to be investigated further if it turns out having this would cause issues.